### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by rhicora**
**Description of changes:** 

This adds a pull request template in accordance with our new developer code of conduct.

**Checklist before we merge your thing:**

- [x] The author is ready to merge
- [x] All changes are described above
- [x] 24 hours for comment have passed, or this is a bugfix
- [x] Hosted and tested; it compiles and works

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Description of changes:**
+
+
+
+**Checklist before we merge your thing:**
+- [ ] The author is ready to merge
+- [ ] All changes are described above
+- [ ] 24 hours for comment have passed, or this is a bugfix
+- [ ] Hosted and tested; it compiles and works
```
Comment (UndeadNappist) [inline pull_request_template.md]: Split "compile" and "tested" into two different checks.

Comment (rhicora) [reply pull_request_template.md]: If it was tested then it compiles.

Comment (UndeadNappist) [reply pull_request_template.md]: Sure, but I compile without testing on occasion.

Comment (rhicora) [reply pull_request_template.md]: Then it's not Hosted and Tested ^.-

Comment (UndeadNappist) [reply pull_request_template.md]: Bahh!

**Pull request merged**